client/my-sites/earn/home.tsx

```diff
@@ -68,7 +68,7 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 					action: { url: supportLink, onClick: () => trackCtaButton( 'simple-payments' ) },
 			  }
 			: {
-					text: translate( 'Upgrade to a Premium Plan!' ),
+					text: translate( 'Upgrade to a Premium Plan' ),
```

Comment:
:disappointed: 23 existing translations will be lost with this change.

Reply:
It's ok. It looks inconsistent with the other buttons if we don't change this.